src/extensions/extensions-api.ts

```diff
@@ -16,14 +16,22 @@
  */
 
 /**
- * SettableProcessingState represents all the Processing states that can be set on an ExtensionInstance's runtimeData.
+ * SettableProcessingState represents all the processing states that can be set
```

Comment:
I think we can remove `SettableProcessingState` from here and just say `Represents all the Processing states...`
```suggestion
 * Represents all the processing states that can be set
```